.eslintignore

```diff
@@ -0,0 +1 @@
+hardhat.config.ts
```

Comment:
Why?

Reply:
The config file has `require("dotenv").config();` which the eslint considers as an error.

Reply:
Then change the require to import.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-example/pull/74/commits/14dca5d1bc47a34696a306f36f335d59a0a50921